eox_tenant/edxapp_wrapper/auth.py

```diff
@@ -0,0 +1,15 @@
+"""
+Auth definitions.
+"""
+
+from importlib import import_module
+from django.conf import settings
+
+
+def get_edx_auth_failed():
+    """ Gets GeneratedCertificate model. """
```

Comment:
```suggestion
    """ Return GeneratedCertificate model. """
```

Reply:
That's a previous commit, I am going to squash